readthedocs/proxito/views/serve.py

```diff
@@ -368,23 +368,35 @@ def get(self, request, proxito_path, template_name='404.html'):
                         project=final_project,
                         version=version,
                         path=path,
+                        full_path=proxito_path,
                     )
                     return resp
 
         self._register_broken_link(
             project=final_project,
             version=version,
             path=path,
+            full_path=proxito_path,
         )
         raise Http404('No custom 404 page found.')
 
-    def _register_broken_link(self, project, version, path):
-        PageView.objects.register_page_view(
-            project=project,
-            version=version,
-            path=path,
-            status=404,
-        )
+    def _register_broken_link(self, project, version, path, full_path):
+        try:
+            PageView.objects.register_page_view(
+                project=project,
+                version=version,
+                path=path,
+                full_path=full_path,
+                status=404,
+            )
+        except Exception:
+            # Don't break doc serving if there was an error
+            # while recording the broken link.
+            log.exception(
+                "Error while recording the broken link",
+                project=project.slug,
```

Comment:
```suggestion
                project_slug=project.slug,
```

Keep compatibility with other keys.